package/python-pint-models.spec

```diff
@@ -37,7 +41,7 @@ BuildArch:      noarch
 %python_subpackages
```

Comment:
I'd remove `%python_subpackages` as we don't want `python${py_version}-pint-models` sub packages. The macro doesn't actually seem to generate them in this setup, but still.

Reply:
Ah, I missed that `%{python_files}` was set. If you want the binary package to be called `python311-pint-models` then yes, it is required. I thought we were generally going for `python-pkg-name` for the single version builds like the metadata packages (which has the advantage of the package name staying the same when the distro primary version is changed), but that's fine too.